featurebyte/service/feature_materialize.py

```diff
@@ -23,6 +33,8 @@
 from featurebyte.service.online_store_table_version import OnlineStoreTableVersionService
 from featurebyte.session.base import BaseSession
 
+FEATURE_TIMESTAMP_COLUMN = "feature_timestamp"
```

Comment:
Do we need to worry about the column name collision here?

Reply:
Good point, will add the prefix to avoid name collision.